egs/wsj/s5/utils/make_lexicon_fst_silprob.pl

```diff
@@ -25,12 +25,24 @@
 # the make_lexicon_fst.pl script except for the word-dependent silprobs part
 
 if (@ARGV != 4) {
-  print STDERR "Usage: $0 lexiconp_disambig.txt \\\n";
-  print STDERR "       silprob.txt silphone sil_disambig_sym > lexiconfst.txt \n";
+  print STDERR "Usage: $0 lexiconp_silprob_disambig.txt \\\n";
+  print STDERR "       silprob.txt silphone_string sil_disambig_sym > lexiconfst.txt \n";
   print STDERR "\n";
   print STDERR "This script is almost the same as the utils/make_lexicon_fst.pl\n";
   print STDERR "except here we include word-dependent silence probabilities\n";
-  print STDERR "when making the lexicon FSTs.\n";
+  print STDERR "when making the lexicon FSTs. ";
+  print STDERR "For details, see paper \nhttp://danielpovey.com/files/2015_interspeech_silprob.pdf\n\n";
+  print STDERR "The lexiconp_silprob_disambig.txt file should have each line like \n";
+  print STDERR "(pronunciation would have to include disambiguation symbols)\n\n";
+  print STDERR "word p(pronunciation|word) p(sil after|word) correction-factor-for-sil ";
```

Comment:
I thought I just listed the paper link in the description there since it's not easy to explain the 2 factors with just a few words...
I will add some descriptions.